flake.nix

```diff
@@ -91,18 +91,23 @@
               runHook preInstall
 
```

Comment:
I could get this to work only if numenc was already in PATH by running `nix shell -v --builders github:Lykos153/numen-nix/fix/numenc` first. We might need to patch with:

```
substituteInPlace phrases/* \
  --replace numenc "$out/bin/numenc"
```